cpp/include/cuspatial/experimental/detail/bounding_box.cuh

```diff
@@ -46,27 +60,32 @@ struct box_minmax {
 
 }  // namespace detail
 
-template <typename IdInputIt, typename PointInputIt, typename BoundingBoxOutputIt>
-BoundingBoxOutputIt trajectory_bounding_boxes(IdInputIt ids_first,
-                                              IdInputIt ids_last,
-                                              PointInputIt points_first,
-                                              BoundingBoxOutputIt bounding_boxes_first,
-                                              rmm::cuda_stream_view stream)
+template <typename IdInputIt, typename PointInputIt, typename BoundingBoxOutputIt, typename T>
+BoundingBoxOutputIt point_bounding_boxes(IdInputIt ids_first,
+                                         IdInputIt ids_last,
+                                         PointInputIt points_first,
+                                         BoundingBoxOutputIt bounding_boxes_first,
+                                         T expansion_radius,
```

Comment:
Discussed offline with @isVoid. Doing this would require doing the addition of the expansion radius to each bounding box after the reduction. We looked at a few of approaches.

1. `thrust::transform` after the `reduce_by_key`. This incurs an additional kernel launch. The cost of this may be larger than the cost of a couple of adds to each input coordinate, since the reduction should be bandwidth bound (so the adds are "free").
2. Use a `transform_output_iterator`. This doesn't work because the output type of a `reduce_by_key` bust be constructable from an instance of the input type. 
3. Leave the code as is (expand every point into a box using the expansion radius).

We chose option 3.